From 7873532368f67253190d32e39a70fa3388355d52 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Mon, 29 Jan 2018 14:35:11 -0600
Subject: [PATCH] Redirect all stderr from installed tests to stdout

The previous fix (db6ed9ede3a3ad5f8ff69e0e2cdb9872ae0f7775) didn't
do a good enough job and autopkgtest was still failing on another
individual test.

This makes sure that stderr will never conflict with autopkgtest
in the future.
---
 data/installed-tests/fwupdmgr.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/installed-tests/fwupdmgr.sh b/data/installed-tests/fwupdmgr.sh
index da69d58..223531e 100755
--- a/data/installed-tests/fwupdmgr.sh
+++ b/data/installed-tests/fwupdmgr.sh
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+exec 2>&1
 dirname=`dirname $0`
 
 # ---
-- 
2.7.4

